src/accelerate/test_utils/testing.py

```diff
@@ -64,7 +64,7 @@ def get_backend():
     elif is_cuda_available():
         return "cuda", torch.cuda.device_count(), torch.cuda.memory_allocated
     elif is_mps_available(min_version="2.0"):
-        return "mps", 1, torch.mps.current_allocated_memory()
+        return "mps", 1, torch.mps.current_allocated_memory
```

Comment:
Nice catch. But doesn't this mean that the return values below `"mps", 1, 0` (for old mps) and `"cpu", 1, 0` (for CPU) are also broken? Should they return `lambda: 0` instead?

Reply:
Agreed, we should use a `lambda` for those

Reply:
Yes, you're right. Added `lambda: 0`. Thanks for pointing that out!